feat: added admin module

NilanjanDaw opened the pull request:
bug: need to add authentication

NilanjanDaw committed:
feat: added admin module

bug: need to add authentication

oyeb commented on app.js:
```diff
@@ -4,8 +4,50 @@ var cookieParser = require('cookie-parser');
+const AdminBro = require('admin-bro')
+const AdminBroSequelize = require('admin-bro-sequelizejs')
+const AdminBroExpress = require('admin-bro-expressjs')
+const bcrypt = require('bcrypt')
```
This is a nitpick: recommend using [agon2](https://www.npmjs.com/package/argon2)

oyeb commented on models/user.js:
```diff
@@ -0,0 +1,13 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
+    username: DataTypes.STRING,
+    email: DataTypes.STRING,
```
Little bit validation:
```js
    email: {
      type: DataTypes.STRING,
      allowNull: false,
      validate: {
        isEmail: true
      }
```

oyeb commented on package.json:
```diff
@@ -9,11 +9,17 @@
     "dotenv": "^8.2.0",
     "ejs": "^3.0.1",
     "express": "~4.16.1",
+    "express-formidable": "^1.2.0",
```
Doesn't seem to be used.

NilanjanDaw replied:
Formidable is used by the admin panel as an external dependency

shinjanxp commented on app.js:
```diff
@@ -4,8 +4,50 @@ var cookieParser = require('cookie-parser');
+    databases: [models],
+    rootPath: '/admin',
+    branding: {
+        companyName: 'PPE Help'
```
**COVID-19 PPE Tracker** please

NilanjanDaw committed:
Bug fix to PR #15 Issue #8

-- Added validation to email field of User table
-- Updated hashing from bcrypt to argon
-- Updated Admin panel name

shinjanxp reviewed (approved):
Looks good to me. @oyeb shall we merge?

NilanjanDaw committed:
Added authentication to admin panel Issue #8

oyeb reviewed (approved):
:+1:
